pkg/innerring/processors/netmap/nodevalidation/locode/validator.go

```diff
@@ -0,0 +1,67 @@
+package locode
+
+// Prm groups the required parameters of the Validator's constructor.
+//
+// All values must comply with the requirements imposed on them.
+// Passing incorrect parameter values will result in constructor
+// failure (error or panic depending on the implementation).
+type Prm struct {
+	// NeoFS LOCODE database interface.
+	//
+	// Must not be nil.
+	DB DB
+}
+
+// Validator is an utility that verifies and updates
+// node attributes associated with its geographical location
+// (LOCODE).
+//
+// For correct operation, Validator must be created
+// using the constructor (New) based on the required parameters
+// and optional components. After successful creation,
+// the Validator is immediately ready to work through API.
+type Validator struct {
+	db DB
+
+	mAttr map[string]attrDescriptor
+}
+
+// TODO: define constants in API lib.
+const (
+	attrKeyLocode = "Locode"
```

Comment:
Most docs I've seen spell `LOCODE` or `UN/LOCODE` in all caps. I think storage node attribute should look the same: `/LOCODE:RU LED`

Thoughts? @cthulhu-rider @realloc

Reply:
I just followed current API [constant](https://github.com/nspcc-dev/neofs-api/blob/113c7d97d1951d140c26509000b6ff8df7d1ec64/netmap/types.proto#L153). It seems like nothing is blocking to make an uppercase constant.

Reply:
We decided with @realloc  and @alexvanin to rename `Locode` -> `UN-LOCODE`.